embed.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func embedSchema(defs Definitions, startingTypes map[string]bool) {
+	for name := range startingTypes {
+		embedDefinition(defs[name], defs)
+	}
+}
+
+func embedDefinition(def *Definition, refs Definitions) {
+	if def == nil {
+		return
+	}
+
+	if len(def.Ref) > 0 {
+		refName := strings.TrimPrefix(def.Ref, defPrefix)
+		if strings.Contains(refName, "RollingUpdate") {
+			fmt.Println("hahahahaha")
```

Comment:
It is good to have `TODO`s here :)

Reply:
I was for debugging.
I have dropped it now.
Thanks for catching it.